protocols/discord.py

```diff
@@ -268,6 +283,37 @@ def _burst_new_client(self, guild, member, pylink_netobj):
         self._update_user_status(guild.id, uid, member.user.presence)
         return pylink_user
 
+    def _burst_webhook_agent(self, webhook_agent, guild, pylink_netobj):
+        try:
+            webhook_agent = split_hostmask(webhook_agent)
+        except ValueError:
+            webhook_agent = [webhook_agent, 'webhooks', 'discord/webhooks-agent']
+        agent_uid = pylink_netobj.uidgen.next_uid()
+        pylink_netobj.users[agent_uid] = pylink_user = User(pylink_netobj, nick=webhook_agent[0],
+                                                            ident=webhook_agent[1], realname=webhook_agent[0],
```

Comment:
I would define the realname as a constant that's more verbose, e.g. "pylink-discord webhooks agent"